src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_3001_4000.cpp

```diff
@@ -4994,61 +4994,6 @@ GUI_TEST_CLASS_DEFINITION(test_3809) {
     GTUtilsTaskTreeView::waitTaskFinished(os);
 }
 
-GUI_TEST_CLASS_DEFINITION(test_3813) {
```

Comment:
Why this test is removed? Which code in UGENE does not work after changes in this patch?

Reply:
Unload option for documents removed from ugene. 
ProjectTreeController.cpp
![image](https://user-images.githubusercontent.com/17490668/146051166-4d4ad2f2-5471-424d-bff1-f6a5624d513c.png)

Reply:
This test also failed on my machine on latest master with corresponding error ` GT_FAIL: (action != nullptr) for 'GTMenu __ clickMenuItem _ action not found for item action_project__unload_selected_action'`

Reply:
This action is present in UGENE only in GUI test mode.
Run it like 'ugeneui --gui-test=test-name' from the command line.